utils/collector/collector.go

```diff
@@ -0,0 +1,39 @@
+// Copyright © 2021 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package collector
+
+import "fmt"
+
+type Collector interface {
+	// Send git package;download common file work as wget or curl;copy local file to dst.
+	Send(buildContext, src, savePath string) error
```

Comment:
Maybe rename function name to Copy is better?

Reply:
`Collect` ? ssh interface already have `Copy(host, srcFilePath, dstFilePath string) error`

Reply:
Collet is fine